components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -30,23 +30,28 @@ typedef NS_ENUM(NSUInteger, MDCTabBarViewLayoutStyle) {
    the leading edge of the bar. */
   MDCTabBarViewLayoutStyleScrollable = 1,
 
+  /**
+   The same as MDCTabBarViewLayoutStyleScrollable, but the selected tab is centered within the bar
+   if its position in the scrollview's content area permits it.*/
+  MDCTabBarViewLayoutStyleScrollableCentered = 2,
```

Comment:
What made you put this as `2`, I think we could run into issues if we are changing the number associated with the enum values for the other ones.

Reply:
I wanted it to go next to `MDCTabBarViewLayoutStyleScrollable`, since it's the same thing but with `Centered` at the end. I did check internally and it looks like nobody is doing anything silly like comparing raw values, but I do see your point. I did mention this in the PR description. I could change it if you feel strongly about it. It's definitely something that should set off alarm bells.

Reply:
This class is used very rarely internally.

Reply:
We tend not to change num values because they may be used in archiving unarchiving. Lets add this one to the end and give it a number we haven't used before.

Reply:
Done.